notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -416,26 +385,26 @@
       "source": [
         "## Introduction to Vertex AI Model Monitoring\n",
         "\n",
-        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skew and drift detection of the features in the inbound prediction requests or skew and drift detection of the feature attributions (Explainable AI) in the outbound prediction response -- that is, the distribution of the attributions on how they contributed to the output (predictions).\n",
+        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skewness and drift detection of the features in the inbound prediction requests or the feature attributions (Explainable AI) in the outbound prediction response. In other words, you monitor the distribution of the attributions that quantify feature contributions to the output (predictions).\n",
         "\n",
         "The following are the basic steps to enable model monitoring:\n",
         "\n",
-        "1. Deploy a `Vertex AI` AutoML or custom tabular model to an `Vertex AI Endpoint`.\n",
+        "1. Deploy a Vertex AI AutoML or custom tabular model to a Vertex AI endpoint.\n",
         "2. Configure a model monitoring specification.\n",
-        "3. Upload the model monitoring specification to the `Vertex AI Endpoint`.\n",
-        "4. Upload or automatic generation of the `input schema` for parsing.\n",
-        "5. For feature skew detection, upload the training data for automatic generation of the feature distribution.\n",
-        "6. For feature attributions, upload corresponding `Vertex AI Explainability` specification.\n",
+        "3. Upload the model monitoring specification to the Vertex AI endpoint.\n",
+        "4. Upload schema or use automatic generation of the *input schema* for parsing.\n",
+        "5. For feature skewness detection, upload the training data. This enables automatic generation of the feature distributions.\n",
+        "6. For feature attributions, upload corresponding *Vertex Explainable AI* specification.\n",
```

Comment:
recommend changing:
- For feature attributions, upload corresponding *Vertex Explainable AI* specification.
TO:
- For feature attributions, upload the corresponding *Vertex Explainable AI* specification.